main_window/main_window.py

```diff
@@ -275,9 +275,18 @@ def init_actuator_valve_label(self):
 
     def init_sensor_reading_label(self):
         self.sensors ={}
-        self.sensors[4] = SensorLabel("Tank Mass", "0", 4, 0, self.ui.sensorLayout)
+        #To increase the font a tad bit 
+        tankMassSensor = SensorLabel("Tank Mass", "0 kg", 4, 0, self.ui.sensorLayout)
+        tankMassSensor.qName.setStyleSheet("font-size: 20px")
+        tankMassSensor.qReading.setStyleSheet("font-size: 20px; font-weight: bold")
```

Comment:
I think increasing the size of the text on these labels is a good idea, can you add this for the other labels as well?